app/javascript/modCenter/singleArticle/util.js

```diff
@@ -0,0 +1,35 @@
+const months = [
```

Comment:
I don't think this should be a blocker but I don't think we need any of these functions. 

JavaScript has a bunch of methods to format time. We should also use the `<time>` element to make time "semantic".

You can check https://github.com/thepracticaldev/dev.to/blob/master/app/assets/javascripts/utilities/localDateTime.js for inspiration (and add to those if you want) and this is how the `<time>` element works: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time

This is also an example of a live `<time>` element in the site: https://github.com/thepracticaldev/dev.to/blob/a9c18b4359c5aa242cadb59d0b5ffa301f71f53a/app/views/comments/_comment_date.erb#L3-L5

Reply:
Thanks Rhymes! I think we can incorporate your suggestions in this PR 👍

Reply:
This has been address @rhymes thanks!